aten/src/ATen/native/cuda/UnaryOpsKernel.cu

```diff
@@ -32,6 +32,14 @@ void logical_not_kernel_cuda(TensorIterator& iter) {
   });
 }
 
+void ceil_kernel_cuda(TensorIterator& iter) {
+  AT_DISPATCH_FLOATING_TYPES_AND_HALF(iter.dtype(), "ceil_cuda", [&]() {
+    gpu_kernel(iter, []GPU_LAMBDA(scalar_t a) -> scalar_t {
+      return std::ceil(a);
```

Comment:
`std::ceil` operates with `double` we need to use `ceilf` for floats for performance reasons.

Reply:
I thought `std::ceil` is overloaded for different floating point types? See https://en.cppreference.com/w/cpp/numeric/math/ceil

Reply:
Ah, right, overloaded in C++11. Good point.